DESCRIPTION

```diff
@@ -45,11 +45,18 @@ Imports:
     tibble,
     purrr,
     rlang,
-    rprojroot
+    rprojroot,
+    usethis,
+    fs,
```

Comment:
The only reason I used {fs} is that it provided an easy way to 'touch' files, `fs::file_touch`, which was used to modify `Makevars`, triggering recompilation. This is no longer needed, so {fs} can be safely removed. I suggest doing it at a later phase when other parts of `rextendr::document()` are finalized & tested.

{usethis} is a more complicated topic. There are definitely some useful functions, but we can have simple replacements. If we position {rextendr} as {usethis} for extendr, perhaps having {usethis}-like behavior and {usethis} as dependency is not that bad. It is not a problem to remove it though, so it is up to you.

Reply:
My thinking is that maybe at some point in the future {usethis} will provide {rextendr} functionality by calling {rextendr} functions. I don't like circular dependencies, so I'd rather not have {rextendr} depend on {usethis}.

Reply:
I understand this, we can certainly get rid of {usehis}. 
I have much less experience in serious package development, but this idea seems counter-intuitive to me.
Ideally, `rextendr::use_extendr` should be pulled into {usethis}, it has no dependencies and simply writes a couple of files. In this case, it is perfectly reasonable to communicate with the user using {usethis} functions, specifically designed for that. 
If this happens, then `rextendr::use_extendr` can call `usethis::use_extendr`, as pretty much a lot of similar packages do.
Basically, I am imagining a sort of inverted dependency here. 
In addition, you suggest to ask {usethis} maintainers to introduce an extra dependency on {rextendr} just to call one function, which I doubt they would like. Not every {usethis} consumer will use extendr, but most of {rextendr} consumers will likely use {usethis} to set up and develop their packages, so it feels more natural for {rextendr} to depend on {usethis} (and make use of it), and not vice verse.

Reply:
I have no strong opinion here, but I feel it's fair to have usethis as a dependency. Since we follow what usethis does and assumes the user is already familiar with usethis-like workflow, rextendr shares its fate with usethis to some extent.

Reply:
See my other comment here: https://github.com/extendr/rextendr/pull/67#discussion_r593830791

In brief, I'm fine with a usethis dependency if it solves a real problem that would be difficult to solve otherwise.

Reply:
I see.